requirements.in

```diff
@@ -0,0 +1,2 @@
+Jinja2>=2.3
```

Comment:
Any reason, why I need to have pip-tools (aka 'pip-tools`) installed to be able to update the `requirements.txt` file from now on?